airflow/hooks/druid_hook.py

```diff
@@ -54,12 +54,12 @@ def get_conn(self):
             conn.extra_dejson.get('endpoint', ''))
 
     @property
-    def ingest_post_url(self):
+    def ingest_post_url(self, protocol='http'):
         conn = self.get_connection(self.druid_ingest_conn_id)
         host = conn.host
         port = conn.port
         endpoint = conn.extra_dejson.get('endpoint', '')
-        return "http://{host}:{port}/{endpoint}".format(**locals())
+        return "{protocol}://{host}:{port}/{endpoint}".format(**locals())
```

Comment:
@hughhhh , the connection table in airflow post gres has a conn_type field. I believe thats the column to represent the protocol . Can we reuse that ?